### PR TITLE
demonstrates best practices for group vars

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
Per discussion in Durham last week, recommend putting group variables for network device connections into [group:vars] sections in inventory rather than in `group_vars/group.yml` files. 

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
- Network Documentation

##### ANSIBLE VERSION
2.5 and higher

### CODE REVIEW

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 1]
```diff
@@ -45,75 +45,84 @@ The examples on this page use the following structure:
...
-Inventory
----------
+Inventory, Connections, Credentials: Grouping Devices and Variables
+-------------------------------------------------------------------
 
 An ``inventory`` file is an INI-like configuration file that defines the mapping of hosts into groups.
 
 In our example, the inventory file defines the groups ``eos``, ``ios``, ``vyos`` and a "group of groups" called ``switches``. Further details about subgroups and inventory files can be found in the :ref:`Ansible inventory Group documentation <subgroups>`.
 
+Because Ansible is a flexible tool, there are a number of ways to specify connection information and credentials. We recommend using the ``my_group:vars`` capability in your inventory file:
```
Comment (gundalow) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]: Suggestion:
```
``[my_group:vars]``
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 2]
```diff
@@ -45,75 +45,84 @@ The examples on this page use the following structure:
...
--------------------------------------
-
-Because Ansible is a flexible tool, there are a number of ways to specify connection information and credentials. We recommend using ``group_vars``.
+.. code-block:: ini
 
-**group_vars/eos.yml**
+   [all:vars]
+   # these defaults can be overridden for any group in the group:vars section
+   ansible_connection=network_cli
+   ansible_python_interpreter=python
```
Comment (acozine) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]: per Peter, get rid of this

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst (hunk 1 above)
Comment (acozine) [reply docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]: Thanks - fixed in c675cf385d54211154d52c0fd95e98bc2c0584ac

**Comment by acozine** [comment]
Given the difficulty of backporting changes to 2.5, I'm going to target 2.6 for this change.

**Review by dharmabumstead — APPROVED** [review]

**Pull request merged**